person.rb

```diff
@@ -2,14 +2,20 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @parent_permission = parent_permission
     @name = name
     @age = age
     @id = Random.rand(1..100)
+    @rentals = []
+  end
+
+  def rent_book(book, date)
+    rental = Rental.new(self, book, date)
+    rentals << rental
   end
```

Comment:
- [ ] Kindly note that creating a rental object on line 17 already adds the rental to the respective `rentals` arrays of book and person, so we don't need line 18.

- [ ] Kindly also maintain the order of the arguments as defined in the Rental `initialize` method.

  ```
  def rent_book(book, date)
    Rental.new(date, book, self)
  end
  ```